cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -152,6 +153,14 @@ public class KafkaClusterTest {
                                     .withPort(9093)
                                     .withType(KafkaListenerType.INTERNAL)
                                     .withTls(true)
+                                    .build(),
+                            new GenericKafkaListenerBuilder()
+                                    .withName("lb")
+                                    .withPort(9094)
+                                    .withType(KafkaListenerType.LOADBALANCER)
+                                    .withNewConfiguration()
+                                        .withAllocateLoadBalancerNodePorts(false)
+                                    .endConfiguration()
                                     .build())
```

Comment:
I don't think you should add it here and have it in all tests. Add it only to the Kafka CR used in the unit test focusing on this.